client/blocks/reader-avatar/style.scss

```diff
@@ -27,4 +30,25 @@
 			background-color: $white;
 		}
 	}
+
+	&.is-compact {
+		&.has-site-and-author-icon {
+			.site-icon {
+				height: 32px;
+				width: 32px;
+
+				.gridicon {
+					height: 32px;
+					width: 32px;
+				}
+			}
+
+			.gravatar {
+				height: 24px;
```

Comment:
discussed offline.  top/left etc should only be tabbed extra when directly below a `postion`